Intersect.Client/Entities/Events/Event.cs

```diff
@@ -77,6 +77,11 @@ public override bool Update()
 
         protected bool TryEnsureTexture(out GameTexture texture)
```

Comment:
Dont know if thats the best solution for this.
Actually this method seems a bit confusing, but as it changes the event texture, sound appropriated.

Reply:
@lucas-tardivo When the graphic is changed, is it being changed via `Event`'s `Load(EntityPacket)` method (L59)?

If so, we can expand the `Graphic` property's setter to do what you are doing here to set the sprite, but at the time of change instead of checking and re-setting it every time `TryEnsureTexture(out GameTexture)` is called.

Reply:
> 

I tried that first, but it does'nt work due to the inheritance from Entity class :(